### PR TITLE
CLDR-16667 Add four kxv locales

### PR DESCRIPTION
**Pull request by DavidLRowe**
CLDR-16667

- [ ] This PR completes the ticket.

This PR handles part of the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by DavidLRowe** [comment]
Note that this PR is NOT ready for review. The exemplar data for kxv.xml and kxv_Orya.xml needs additional information. However, I want to make sure that there are no other problems pending.

File: common/supplemental/likelySubtags.xml [hunk 1]
```diff
@@ -6483,9 +6485,6 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<likelySubtag from="kxq" to="kxq_Latn_ID" origin="sil1"/>	<!-- Smärky Kanum ➡︎ Smärky Kanum (Latin, Indonesia) -->
 		<likelySubtag from="kxr" to="kxr_Latn_PG" origin="sil1"/>	<!-- Koro [Papua New Guinea] ➡︎ Koro [Papua New Guinea] (Latin, Papua New Guinea) -->
 		<likelySubtag from="kxt" to="kxt_Latn_PG" origin="sil1"/>	<!-- Koiwat ➡︎ Koiwat (Latin, Papua New Guinea) -->
-		<likelySubtag from="kxv" to="kxv_Orya_IN" origin="sil1"/>	<!-- Kuvi ➡︎ Kuvi (Odia, India) -->
-		<likelySubtag from="kxv_Latn" to="kxv_Latn_IN" origin="sil1"/>	<!-- Kuvi (Latin) ➡︎ Kuvi (Latin, India) -->
-		<likelySubtag from="kxv_Telu" to="kxv_Telu_IN" origin="sil1"/>	<!-- Kuvi (Telugu) ➡︎ Kuvi (Telugu, India) -->
```
Comment (srl295) [inline common/supplemental/likelySubtags.xml:6488]: Maybe need this back as well as Orya

**Review by srl295 — APPROVED** [review]

**Comment by DavidLRowe** [comment]
I think I was at this point previously. Errors now are:
```
Error:  (TestBasic.java:829)  Error: Locale has children but is missing default contents locale: kxv, children: [kxv_Latn]; possible fixes for children:
Error:  (TestBasic.java:829)  Error: Locale has children but is missing default contents locale: kxv_Latn, children: [kxv_Latn_IN]; possible fixes for children:
```
but nothing listed for "possible fixes".

**Comment by srl295** [comment]
Can you file a bug on the err message? It should be more helpful. I suspect
we need an overhaul of how multi script locales are added.

File: common/supplemental/supplementalMetadata.xml [hunk 2]
```diff
@@ -1817,7 +1817,8 @@ For terms of use, see http://www.unicode.org/copyright.html
 			ja_JP jbo_001 jgo_CM jmc_TZ jv_ID
 			ka_GE kab_DZ kaj_NG kam_KE kcg_NG kde_TZ kea_CV ken_CM kgp_BR khq_ML ki_KE
 			kk_KZ kkj_CM kl_GL kln_KE km_KH kn_IN ko_KR kok_IN kpe_LR ks_Arab ks_Arab_IN
-			ks_Deva_IN ksb_TZ ksf_CM ksh_DE ku_TR kw_GB ky_KG
+			ks_Deva_IN ksb_TZ ksf_CM ksh_DE ku_TR kw_GB
+			kxv_IN kxv_Deva_IN kxv_Telu_IN kxv_Orya_IN ky_KG
```
Comment (srl295) [inline common/supplemental/supplementalMetadata.xml:1821]:
```suggestion
			kxv_Latn kxv_Deva_IN kxv_Telu_IN kxv_Orya_IN ky_KG
```

trying this…

**Comment by DavidLRowe** [comment]
@srl295 Errors now are:
```
Error:  (TestBasic.java:829)  Error: Locale has children but is missing default contents locale: kxv_Latn, children: [kxv_Latn_IN]; possible fixes for children:
Error:  (TestInheritance.java:594)  Error: default contents inconsistent with likely subtag: (c)
	kxv => (ls) kxv_Latn_IN
	kxv => (dc) kxv_Latn
	kxv_Latn => (dc) null
Error:  (TestInheritance.java:490)  Error: Missing default content for: kxv_Latn
```

**Comment by srl295** [comment]
> @srl295 Errors now are:
> 
> ```
> 
> Error:  (TestBasic.java:829)  Error: Locale has children but is missing default contents locale: kxv_Latn, children: [kxv_Latn_IN]; possible fixes for children:
> 
> Error:  (TestInheritance.java:594)  Error: default contents inconsistent with likely subtag: (c)
> 
> 	kxv => (ls) kxv_Latn_IN
> 
> 	kxv => (dc) kxv_Latn
> 
> 	kxv_Latn => (dc) null
> 
> Error:  (TestInheritance.java:490)  Error: Missing default content for: kxv_Latn
> 
> ```

OK… I will keep looking

**Comment by DavidLRowe** [comment]
Following example of "az", added kxv_Latn_IN to @locales in supplementalMetadata.xml.

**Comment by DavidLRowe** [comment]
@srl295 This time for sure, Rocky! (But maybe you're too young to remember the old Bullwinkle cartoons!)

**Comment by srl295** [comment]
> @srl295 This time for sure, Rocky! (But maybe you're too young to remember the old Bullwinkle cartoons!)

Watch me pull a rabbit out of my hat…

**Review by srl295 — APPROVED** [review]
LGTM

**Comment by srl295** [comment]
@DavidLRowe  you'll need to hit the Squash button otherwise great!

**Comment by DavidLRowe** [comment]
@srl295 All my mistakes have been squashed away. Can you approve it before the tests finish? 

But, IIUC there'll need to be a "push to production" before these locales will be available in Survey Tool, right?

**Review by srl295 — APPROVED** [review]

**Comment by srl295** [comment]
> @srl295 All my mistakes have been squashed away. Can you approve it before the tests finish? 
> 
> 
> 
> But, IIUC there'll need to be a "push to production" before these locales will be available in Survey Tool, right?

I'm sure there will be more pushes but yes

**Pull request merged**